### ISSUE TITLE
When installing, "Cannot copy 'share#foo'" 

### ISSUE THREAD

**Issue by eliot1785** [plack/Plack#600]
When installing on Strawberry Perl 64-bit, Windows 7, I encountered a file copy error. Looking at the actual error, there's a # within the file name, but other than that I'm not sure what to look for here. Any suggestions would be welcome. Is Plack supported on Windows?

```
t/10_oo.t ...................... ok
t/11_net_empty_port.t .......... 2/?     # found an empty IPv6 port
t/11_net_empty_port.t .......... ok
t/12_pass_wait_port_options.t .. 29/? IPC Fatal Error: Leftover files in the directory (C:\Users\sdewey\AppData\Local\Te
mp\test2~11800~zOi8vf\11800~0~1~-10280~0~13~Test2~Event~Note)!
Could not remove IPC temp dir (C:\Users\sdewey\AppData\Local\Temp\test2~11800~zOi8vf) at C:/Strawberry/perl/lib/Test2/IP
C/Driver/Files.pm line 420 during global destruction.
t/12_pass_wait_port_options.t .. All 51 subtests passed
t/13_undef_port.t .............. 1/22 IPC Fatal Error: Leftover files in the directory (C:\Users\sdewey\AppData\Local\Te
mp\test2~10900~Z2DjXw\10900~0~1~-14600~0~13~Test2~Event~Note)!
Could not remove IPC temp dir (C:\Users\sdewey\AppData\Local\Temp\test2~10900~Z2DjXw) at C:/Strawberry/perl/lib/Test2/IP
C/Driver/Files.pm line 420 during global destruction.
t/13_undef_port.t .............. All 22 subtests passed
t/14_listen.t .................. ok

Test Summary Report
-------------------
t/01_simple.t                (Wstat: 0 Tests: 4 Failed: 2)
  Failed tests:  2, 4
  Parse errors: Tests out of sequence.  Found (2) but expected (3)
                Plan (1..2) must be at the beginning or end of the TAP output
                Bad plan.  You planned 2 tests but ran 4.
t/12_pass_wait_port_options.t (Wstat: 0 Tests: 52 Failed: 1)
  Failed test:  52
  Parse errors: Plan (1..51) must be at the beginning or end of the TAP output
                Bad plan.  You planned 51 tests but ran 52.
t/13_undef_port.t            (Wstat: 0 Tests: 23 Failed: 1)
  Failed test:  23
  Parse errors: Bad plan.  You planned 22 tests but ran 23.
Files=15, Tests=148, 113 wallclock secs ( 0.16 usr +  0.91 sys =  1.06 CPU)
Result: FAIL
Failed 3/15 test programs. 4/148 subtests failed.
gmake.EXE: *** [Makefile:862: test_dynamic] Error 255
  TOKUHIROM/Test-TCP-2.19.tar.gz
  C:\Strawberry\c\bin\gmake.EXE test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports TOKUHIROM/Test-TCP-2.19.tar.gz
  MIYAGAWA/Plack-1.0044.tar.gz
  Has already been unwrapped into directory C:\Users\sdewey\.cpan\build\Plack-1.0044-0
  MIYAGAWA/Plack-1.0044.tar.gz
  Has already been prepared
Running make for M/MI/MIYAGAWA/Plack-1.0044.tar.gz
Warning: Prerequisite 'Test::TCP => 2.15' for 'MIYAGAWA/Plack-1.0044.tar.gz' failed when processing 'TOKUHIROM/Test-TCP-
2.19.tar.gz' with 'make_test => NO'. Continuing, but chances to succeed are limited.
cp share\baybridge.jpg blib\lib\auto\share\dist\Plack\baybridge.jpg
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
ERROR: Cannot copy 'share#foo' to 'blib\lib\auto\share\dist\Plack#foo': No such file or directory
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 at -e line 1.
gmake.EXE: *** [Makefile:1123: config] Error 255
  MIYAGAWA/Plack-1.0044.tar.gz
  C:\Strawberry\c\bin\gmake.EXE -- NOT OK
```

**Comment by charsbar** [comment]
@eliot1785, see https://github.com/plack/Plack/issues/591 for details and workarounds.

**Comment by miyagawa** [comment]
Also fixed in #599
